specification/glossary.md

```diff
@@ -104,6 +104,10 @@ A company or organization that provides outsourced management and support of a r
 
 A term that describes a service that is available and provided immediately or as needed, without requiring a pre-scheduled appointment or prior arrangement. In cloud computing, virtual machines can be created and terminated as needed, i.e. on demand.
 
+<a name="glossary:negotiated-discount"><b>Negotiated Discount</b></a>
+
+A contractual agreement where a customer commits to specific spend or usage goals over a [*term*](#glossary:term) in exchange for discounted rates across varying SKUs.  Unlike [*commitment-based discounts*](#glossary:commitment-based-discount), negotiated discounts are typically more customized to customer's accounts, can be burned down at varying frequencies, and may be overlap with *commitment-based discounts*.
```

Comment:
> Unlike *commitment-based discounts*  ... and may be overlap with *commitment-based discounts*.

This might raise questions for readers. Maybe a separate sentence about commitment-based discounts could clarify, like:

> Negotiated discounts are typically more customized to customer's accounts, can be burned down at varying frequencies. Negotiated discounts may apply to commitment-based discounts.

Reply:
This is the exact FinOps terminology (literally copy and pasted), so going to keep consistent